openshift/template.yaml

```diff
@@ -496,7 +496,7 @@ parameters:
     displayName: ENABLE_OPEN_USER_REGISTRATION
     required: true
     name: ENABLE_OPEN_USER_REGISTRATION
-    value: "true"
+    value: "False"
```

Comment:
This will work however just to mention that the right values are 1 and 0.

https://github.com/pjialin/django-environ/blob/dc2d281e45eb41a723549e01e435ae1fe89b0a96/environ/environ.py#L313